### PR TITLE
You can no longer mount borgs while stunned.

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/tgstation/issues/28128.

holy shit this is an ugly section of code

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -1116,7 +1116,7 @@
 		return
 	if(incapacitated())
 		return
-	if(M.restrained())
+	if(M.restrained() || M.incapacitated())
```
Comment (Cyberboss) [inline code/modules/mob/living/silicon/robot/robot.dm]: incapacitated checks restrained, so remove that bit

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/silicon/robot/robot.dm (hunk 1 above)
Comment (bgobandit) [inline code/modules/mob/living/silicon/robot/robot.dm]: I was wondering about this

**Comment by FantasticFwoosh** [comment]
How about removing borg riding huh for being a inconsistent bug & balance upsetting system.

**Comment by Tacolizard** [comment]
ANYTHING THAT'S ADDED WITHOUT REBALANCING EVERY OTHER FEATURE IN THE GAME IS BAD

**Pull request merged**